.circleci/config.yml

```diff
@@ -602,6 +602,8 @@ jobs:
       - run: docker pull alpine:latest # make sure it's fresh
       - deploy:
           command: |
+            git diff
```

Comment:
what is the reason for this?

Reply:
so that we can tell _why_ the release failed
if it fails due to "there's a change somewhere in your git repo" like last time